internal/dag/gatewayapi_processor.go

```diff
@@ -1429,7 +1429,7 @@ func gatewayPathMatchCondition(match *gatewayapi_v1beta1.HTTPPathMatch, routeAcc
 		return &ExactMatchCondition{Path: path}, true
 	}
 
-	routeAccessor.AddCondition(status.ConditionNotImplemented, metav1.ConditionTrue, status.ReasonPathMatchType, "HTTPRoute.Spec.Rules.PathMatch: Only Prefix match type and Exact match type are supported.")
+	routeAccessor.AddCondition(gatewayapi_v1beta1.ConditionAccepted, metav1.ConditionFalse, gatewayapi_v1beta1.ReasonUnsupportedValue, "Unsupported HTTPRoute path match type.")
```

Comment:
Looks like I gave you the wrong names here in my initial suggestion, it should be:

```suggestion
	routeAccessor.AddCondition(gatewayapi_v1beta1.RouteConditionAccepted, metav1.ConditionFalse, gatewayapi_v1beta1.RouteReasonUnsupportedValue, "Unsupported HTTPRoute path match type.")
```

https://github.com/projectcontour/contour/blob/main/CONTRIBUTING.md will walk you through how to ensure the updated code builds locally and passes all unit tests. I'm guessing there is probably a test case or two to update here.

Reply:
Please also leave the Condition message as it was previously - `HTTPRoute.Spec.Rules.PathMatch: Only Prefix match type and Exact match type are supported.`